packages/SwingSet/docs/delivery.md

```diff
@@ -826,16 +833,17 @@ A Vat might be informed that one Promise has been resolved to another Promise
 ## Sample Message Delivery
 
 This section follows a variety of messages are they are sent from Vat-1 to
-Vat-2.
+Vat-2. Syntax like `foo!bar(baz)` is used to indicate a message to Object or
```

Comment:
I figure this note allows us to keep it here as a not-actually-executable shorthand.

Reply:
Yeah, good point, and I do that all the time in unit test comments. Let's update it to tildot then, so at least we only have one archaic dialect instead of two.